mobify/platform-scaffold
# update empty address and display region instead of regionId

regionId is showing instead of region

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1430

## Changes
- (update empty address and display region instead of regionId)

## Todos

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- login with mobifyqa4@gmail.com to see empty address
- login with mobifyqa3@gmail.com to see region
